### PR TITLE
docs: include hints on specifying a locale can help startup time

### LINKED ISSUE #1114: this package was slowing down my jest tests
I have been debugging my jest tests all day because they were failing due to timeouts, and sometimes passing but it took like 200 seconds sometimes to run few simple tests. Long story short, I have disabled creating my random mock data with faker and used hard-coded data instead. Now my jest runs in few seconds.

Note that it was not slowing down particular unit tests (they took few milliseconds), but the whole jest runtime took over hundreds of seconds. So it does not play nice with jest runner or whatever on windows 10 and also WSL.

### PR DESCRIPTION
**Pull request by conner-c**
Updated: README.md to include a quick message about why loading every locale can slow down startup times.
Updated: localization.md includes more information about the limitations of some locales.
Updated: usage.md includes hints on how to specify a specific locale and why it can help startup times.
closes #1114

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Array.from({ length: 10 }).forEach(() => {
   USERS.push(createRandomUser());
 });
 ```
-
+The above code indicates a basic usage of Faker. The point of interest is the import statements at the top. The first import indicates how one can import the entirety of Faker, which includes every local, while the commented-out import showcases how to import only a single local. In most situations, importing a single local is preferable for performance reasons since loading every local into the memory takes time during start-up. In some instances, however, some locales have limited coverage and, as such, use the English locale as a fallback.
```
Comment (ST-DDT) [inline README.md]:
```suggestion
The above code indicates a basic usage of Faker.
The point of interest is the import statements at the top.
The first import indicates how one can import the entirety of Faker, which includes every locale, while the commented-out import showcases how to import only a single locale.
In most situations, importing a single locale is preferable for performance reasons since loading every locale into the memory takes time during start-up.
In some instances, however, some locales have limited coverage and, as such, use the English locale as a fallback.
```

Could you amend the second last sentence, with a hint like:

> Some test frameworks reload the imports in every test-file, so the increased loading times add up quickly.

I'm not sure about the last sentence. IMO it is confusing, because all locales (currently) use the en locale as fallback, so it is loaded always.

File: docs/guide/localization.md [hunk 2]
```diff
@@ -27,6 +27,9 @@ This will then just load the German locales with additional English locales as f
 The English locales are around 600 KB in size.  
 All locales together are around 5 MB in size.
 :::
+:::tip Note
+Some locales have limited coverage and, as such, rely on English locales for features that the locale does not cover. In most cases, using a specific locale is still preferable to reduce start-up times.
+:::
```
Comment (ST-DDT) [inline docs/guide/localization.md]:
```suggestion

:::tip Note
Some locales have limited coverage and, as such, rely on English locales for features that the locale does not cover.
In most cases, using a specific locale is still preferable to reduce start-up times.
:::
```

Can you rephrase this a bit, because the default faker instances for all locales actually use the en as default.

File: docs/guide/usage.md [hunk 3]
```diff
@@ -6,11 +6,14 @@ Using Faker is as easy as importing it from `@faker-js/faker`.
 
 ```js
 import { faker } from '@faker-js/faker';
-
+// or, if desiring specific locale
```
Comment (ST-DDT) [inline docs/guide/usage.md]:
```suggestion
// or, if desiring only a specific locale
```

File: docs/guide/usage.md [hunk 4]
```diff
@@ -6,11 +6,14 @@ Using Faker is as easy as importing it from `@faker-js/faker`.
...
+// import { faker } from '@faker-js/faker/locale/de'
 const randomName = faker.person.fullName(); // Rowan Nikolaus
 const randomEmail = faker.internet.email(); // Kassandra.Haley@erich.biz
 ```
-
+:::tip Note
+Using the first import statement will load every locale into memory. As such, start-up times and performance may be slow. Thus, by declaring a locale in the import, one can increase performance and reduce the time on start-up.
+:::
```
Comment (ST-DDT) [inline docs/guide/usage.md]:
```suggestion

:::tip Note
Using the first import statement will load every locale into memory.
As such, start-up times and performance may be slow.
Thus, by declaring a locale in the import, one can increase performance and reduce the time on start-up.
:::

```

**Review by ST-DDT — COMMENTED** [review]
Please put long sentences in a new line each.

**Review by conner-c — COMMENTED** [review]
Updated files with given notes. I have separated the sentences into individual lines. I have rephrased sections.

File: README.md [hunk 5]
```diff
@@ -63,8 +63,13 @@ export function createRandomUser(): User {
...
   USERS.push(createRandomUser());
 });
+
 ```
-The above code indicates a basic usage of Faker. The point of interest is the import statements at the top. The first import indicates how one can import the entirety of Faker, which includes every local, while the commented-out import showcases how to import only a single local. In most situations, importing a single local is preferable for performance reasons since loading every local into the memory takes time during start-up. In some instances, however, some locales have limited coverage and, as such, use the English locale as a fallback.
+The above code indicates a basic usage of Faker. 
+The point of interest is the import statements at the top. 
+The first import indicates how one can import the entirety of Faker, which includes every local, while the commented-out import showcases how to import only a single local. 
+In most situations, importing a single local is preferable for performance because some testing frameworks reload imports for every test file, which causes startup latencies to add up quickly.
+Thus, limiting the import to a single locale can speed up startup times.
```
Comment (ST-DDT) [inline README.md:72]:
```suggestion
Thus, limiting the import to a single locale can speed up startup times.

```

File: README.md [hunk 6]
```diff
@@ -63,7 +63,13 @@ export function createRandomUser(): User {
 Array.from({ length: 10 }).forEach(() => {
   USERS.push(createRandomUser());
 });
+
 ```
+The above code indicates a basic usage of Faker.
```
Comment (ST-DDT) [inline README.md]:
````suggestion
```

The above code indicates a basic usage of Faker. 
````

**Review by ST-DDT — DISMISSED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]
This should be very helpful. Thank you.

**Review by ejcheng — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]
